crates/eframe/src/web/mod.rs

```diff
@@ -246,3 +246,43 @@ pub fn percent_decode(s: &str) -> String {
         .decode_utf8_lossy()
         .to_string()
 }
+
+use js_sys::Uint8Array;
+use log::{debug, error};
+use std::sync::Arc;
+use wasm_bindgen::JsCast;
+use wasm_bindgen_futures::JsFuture;
+use web_sys::window;
```

Comment:
Move them to inside the `fn`. That way they will move with the function if it has to move file